src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Repeater/Given_ItemsRepeater.cs

```diff
@@ -484,7 +484,7 @@ public async Task When_ItemSignificantlyHigher_Then_VirtualizeProperly()
 
 			item3UpdatedVerticalOffset.Should().Be(item3OriginalVerticalOffset); // Confirm that item #3 has not been moved down
 			var result = await UITestHelper.ScreenShot(sut.Repeater);
-			ImageAssert.HasColorAt(result, 10, 10, Colors.FromARGB("#008000")); // For safety also check it's effectively the item 3 that is visible
+			ImageAssert.HasColorAt(result, 10, 10 + (int)Math.Ceiling(sut.Scroller.VerticalOffset), Colors.FromARGB("#008000")); // For safety also check it's effectively the item 3 that is visible
```

Comment:
No this does not really test what is expected, if the `result` as changed due to clipping updates, you should change it to :

```suggestion
			var result = await UITestHelper.ScreenShot(sut.Root);
			ImageAssert.HasColorAt(result, 15, 15, Colors.FromARGB("#008000")); // For safety also check it's effectively the item 3 that is visible
```